packages/sidenav/src/index.js

```diff
@@ -76,7 +76,7 @@ export const Sidenav = React.forwardRef(
               WebkitOverflowScrolling: 'touch',
               transition: 'transform .2s ease-out',
               transform: [open ? 'translateX(0)' : 'translate(-100%)', 'none'],
-              bg: ['background', 'transparent'],
+              backgroundColor: ['background', 'transparent'],
```

Comment:
Thanks! But this *should* work as-is and appears to be working on https://theme-ui.com – were you running into issues where this wasn't working?

Reply:
For me, `bg: 'background'` was generating `background: background`.

Reply:
Ah! If you see `background-color: background` being generated, that's usually a case of mis-matched theming context. I'm looking into better warnings when that happens, but if a value is missing from React context, it can cause unexpected issues like this